src/containers/Topmenu/index.jsx

```diff
@@ -17,28 +18,64 @@ const MenuCont = styled.div`
 class Topmenu extends Component {
     state = {
         categories: [],
+        isLoading: true,
     };
 
+    componentWillMount() {
+        localStorage.getItem('categories') &&
+            this.setState({
+                categories: JSON.parse(localStorage.getItem('categories')),
+                isLoading: false,
+            });
+    }
+
     componentDidMount = () => {
-        this.getCategories();
+        let newCategories = this.getCategories();
+        if (!localStorage.getItem('categories')) {
+            newCategories;
+        }
     };
 
     getCategories = async () => {
         const url = BaseApi.getUrl('categories?_limit=4');
         const response = await fetch(url);
         const categories = await response.json();
-        this.setState(() => ({ categories }));
+        this.setState(() => ({ categories, isLoading: false }));
     };
 
+    componentWillUpdate(nextProps, nextState) {
+        localStorage.setItem('categories', JSON.stringify(nextState.categories));
+    }
     render() {
-        return (
-            <Fragment>
-                <Navigation categories={this.state.categories} />
-                <MenuCont>
-                    <CategoryList categories={this.state.categories} />
-                </MenuCont>
-            </Fragment>
-        );
+        const { isLoading } = this.state;
+        if (isLoading) {
+            return (
+                <Fragment>
+                    <SkeletonWrapper width="auto" className="skeleton-phone">
+                        <SkeletonItem width="28px" height="4px" />
+                        <SkeletonItem width="28px" height="4px" />
+                        <SkeletonItem width="28px" height="4px" />
+                    </SkeletonWrapper>
+                    <MenuCont>
+                        <SkeletonWrapper type="row" width={`${500}px`}>
```

Comment:
Aqui pode ser uma string simples mesmo 
```suggestion
                        <SkeletonWrapper type="row" width="500px">
```